naveenG/task2/sample.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>sample</title>
+    
+</head>
+<body>
+
+ 
+ <table id="table" border="">
```

Comment:
Why border is empty? if not needed please remove

Reply:
It is needed Anna , because I given the border size to that table using that border attribute.